doc/services/device_mgmt/ec_host_cmd.rst

```diff
@@ -36,13 +36,43 @@ one backend layer.
 .. image:: ec_host_cmd_shi.png
    :align: center
 
+Another case is SPI. Unfortunately, the current SPI API can't be used to handle the host commands
+communication. The main issues are unknown command size sent by the host(the SPI transaction
+sends/receives specific number of bytes) and need to constant sending status byte(the SPI module
+is enabled and disabled per transaction). It forces implementing the SPI driver within a backend,
+as it is done for SHI. That means a SPI backend has to implemented per chip family. However, it
+can be changed in the future once the SPI API is extended to host command needs.
+
+That approach requires configuring the SPI dts node in a special way. Status of such node needs to
+be set to "disable" not to be handled by a default SPI driver. The rest of properties should be
+configured as usual. Example of the SPI node for STM32:
+
+.. code-block:: devicetree
+
+   &spi1 {
+           pinctrl-0 = <&spi1_nss_pa4 &spi1_sck_pa5 &spi1_miso_pa6 &spi1_mosi_pa7>;
+           pinctrl-names = "default";
+           dmas = <&dma2 3 3 0x38440 0x03>,
+                <&dma2 0 3 0x38480 0x03>;
+           dma-names = "tx", "rx";
+           /* This field is used to point at our CS pin */
+           cs-gpios = <&gpioa 4 GPIO_ACTIVE_LOW>;
+           /* Disable on purpose. This is done to handle the SPI module by the SPI
+              backend, not by the STM32 SPI driver */
+           status = "disabled";
```

Comment:
Setting a node to `disabled` but then using the node is pretty confusing.  Are there any examples in the code doing this?

Since you're implementing a replacement for spi_ll_stm32.c driver, you could have the application replace the `compatible` property for the SPI node in the application overlay.

```
   &spi1 {
           compatible = "st,stm32-spi-host-cmd";
           pinctrl-0 = <&spi1_nss_pa4 &spi1_sck_pa5 &spi1_miso_pa6 &spi1_mosi_pa7>;
           pinctrl-names = "default";
           /* ... */
```

Reply:
>Are there any examples in the code doing this?
No, it was my idea.

Changing the compatible string looks much better. I haven't considered this option. I have to add a new yaml file with a new compatible to generate needed devicetree defines/macros.